src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommandParser.cs

```diff
@@ -42,8 +42,9 @@ internal static class WorkloadInstallCommandParser
         public static readonly Option IncludePreviewOption =
             new Option<bool>("--include-previews", LocalizableStrings.IncludePreviewOptionDescription);
 
-        public static readonly Option FromCacheOption =
-            new Option<string>("--from-cache", LocalizableStrings.FromCacheOptionDescription) { };
+        public static readonly Option FromCacheOption = new Option<string>("--from-cache", LocalizableStrings.FromCacheOptionDescription);
+
+        public static readonly Option DownloadToCacheOption = new Option<string>("--download-to-cache", LocalizableStrings.DownloadToCacheOptionDescription);
```

Comment:
@KathleenDollard, were you going to work on a spec for what the CLI commands for workloads would be?

In this PR, it looks like the following would be how to create and use a cache:

- `dotnet workload install ios android --download-to-cache "c:\netsdkofflinecache"
- `dotnet workload install ios android --from-cache "c:\netsdkofflinecache"